src/component/FooterNav.js

```diff
@@ -0,0 +1,81 @@
+import {AiFillHome} from 'react-icons/ai';
+import {FaHeart} from 'react-icons/fa';
+import {NavLink} from 'react-router-dom';
+import {keyframes} from 'styled-components';
+import styled from 'styled-components';
+
+export default function FooterNav() {
+  return (
+    <NavStyledContainer>
+      <StyledNavLink to="/">
+        <span></span>
+        <StyledHomeIcon />
+      </StyledNavLink>
+
+      <StyledNavLink to="/favorites">
+        <StyledHeardIcon />
+      </StyledNavLink>
```

Comment:
I would suggest to add some accessible names to the links:
```suggestion
      <StyledNavLink aria-label="home" to="/">
        <span></span>
        <StyledHomeIcon />
      </StyledNavLink>

      <StyledNavLink aria-label="favorites" to="/favorites">
        <StyledHeardIcon />
      </StyledNavLink>
```

Reply:
done